doc/getting_started.rst

```diff
@@ -24,6 +24,14 @@ A `template repository <https://github.com/sphinx-gallery/sample-project>`_,
 with sample example galleries and basic configurations is also available to
 help you get started.
 
+.. note::
+   Working `sphinx 
+   builders<https://www.sphinx-doc.org/en/master/man/sphinx-build.html#options>`_ 
+   for sphinx_gallery include `html` and `latex`.
+
+   E.g., `dirhtml` will cause broken image links.
```

Comment:
Code in RST is double-backticks. Single-backticks is "default_role", whatever we have that set to. Probably better just to be explicit with code mode here

Also, I think a missing space is causing the link rendering not to work

https://2561-25860190-gh.circle-artifacts.com/0/rtd_html/getting_started.html

```suggestion
   builders <https://www.sphinx-doc.org/en/master/man/sphinx-build.html#options>`_ 
   for sphinx_gallery include `html` and `latex`. Support for other builders
   is not guaranteed (e.g., `dirhtml` is known to be broken and will cause
   broken image links.).
```